README.md

```diff
@@ -55,61 +57,68 @@ Optional dependencies are always installed but can be omitted by using `npm inst
 
 ### Version 8
 
+#### SQFormScrollableCard no longer accepts titleVariant as a prop
+
+- No changes are required if the consumer application is a JS application
+- Consumer applications that use TypeScript will require this prop to be removed if present to resolve the type error.
```

Comment:
I had considering mentioning that the header height was fixed to 50px. and perhaps that the value is now fixed at h5.

meh, you beat me to it.

Reply:
No worries. I'm attempting to strike a balance between minimal info needed for the migration vs documentation